recipes/cgat-scripts/meta.yaml

```diff
@@ -0,0 +1,207 @@
+
+package:
+  name: cgat-scripts
+  version: 0.2.6
+
+source:
+  url: https://pypi.python.org/packages/fe/86/be2a14843dc272abe33dd0de754e8ba142fc07c3cfe11410be7230d27078/CGAT-0.2.6.tar.gz
+  md5: 44f4abc43180440a22883c0f93312338
+
+build:
+  skip: True # [not py27 or osx]
+  number: 0
+  entry_points:
+    - cgat = CGAT.cgat:main
+
+requirements:
+  build:
+    # system deps
+    - nomkl
```

Comment:
Can these pinnings be relaxed (e.g. to the major versions) or do you definitely want to keep them so strict?